Bottom: a35952ead6921f75b9c971271af3a86b2f2d988d
Top:    d3e806e7c5b03d285376027141d7b6ac499b15dc
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:24:23 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/tasks/main.yml b/roles/buildbot-master/tasks/main.yml
index f2d530d1..b68c44c2 100644
--- a/roles/buildbot-master/tasks/main.yml
+++ b/roles/buildbot-master/tasks/main.yml
@@ -17,19 +17,18 @@
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac
     - master.cfg
     - openembedded.py
     - ptxdist.py
 
 - name: Populate config templates
-  copy:
-    src: "{{ item }}"
+  template:
+    src: "{{ item }}.j2"
     dest: "{{ config_hostdir }}/{{ item }}"
     group: '{{ buildbot_uid }}'
     owner: '{{ buildbot_uid }}'
   loop:
-    - buildbot.tac.j2
+    - buildbot.tac
 
 - name: Create secrets volume
   file:
